examples/CMakeLists.txt

```diff
@@ -67,9 +67,17 @@ IF(WIN32)
   SET(EXAMPLES_INSTALL_BIN examples/bin/)
   SET(EXAMPLES_INSTALL_SRC examples/src/)
 ELSE()
-  SET(EXAMPLES_INSTALL_BIN ${CMAKE_INSTALL_FULL_LIBDIR}/simbody/examples/)
+  SET(EXAMPLES_INSTALL_BIN ${CMAKE_INSTALL_LIBDIR}/simbody/examples/) # if this changes, change the corresponding
+                                                                      # FULL version in file(RELATIVE_PATH ) command
   SET(EXAMPLES_INSTALL_SRC ${CMAKE_INSTALL_DOCDIR}/examples/src/)
-  SET(EXAMPLES_SYMLINK_BIN ${CMAKE_INSTALL_DOCDIR}/examples/)
+  # Use full paths for the following as it will be used to create a symlink
+  # The symlink itself will be a relative path, so the package will be relocatable
+  # Cannot use CMAKE_INSTALL_FULL_DOCDIR as it evaluates to a different value,
+  # as CMAKE_INSTALL_DOCDIR is changed as a "normal" variable, not in CACHE
```

Comment:
Hmmm ... this seems awkward. I see that the problem comes from the main CMakeLists file [here](https://github.com/simbody/simbody/blob/master/CMakeLists.txt#L35). I don't think the issue is cache vs. normal; looks like someone changed `CMAKE_INSTALL_DOCDIR` but not `CMAKE_FULL_INSTALL_DOCDIR`. They should match, I think. One fix might be to change them both. Another might be to change the PROJECT name on non-WIN32 platforms from PROJECT(Simbody) to PROJECT(simbody). The latter might have side effects I suppose. I wonder also how we manage to get lower case `simbody` names elsewhere, like in the `lib` and `libexec` directories?